apps/api/src/app/controllers/command/__snapshots__/command-payload-schemas.spec.ts.snap

```diff
@@ -70,7 +70,7 @@ exports[`command payload schemas Command payload schema should match the snapsho
           },
           {
             "label": "wav",
-            "value": "audio/x-wav",
+            "value": "audio/wav",
```

Comment:
Hopefully there's a cypress test that proves that the client still works. Did you manually try pushing through create media item or another command that uses this on the payload?

Reply:
Here's creating a media item from the CLI:

```
Importing data from file "data/mediaitem-2-command-stream.data.json" into COSCRAD...
Attempting to execute command FSA: {"type":"CREATE_MEDIA_ITEM","payload":{"aggregateCompositeIdentifier":{"id":"dda1df6d-9ee4-4bd9-9a82-3d4bd74bb831","type":"mediaItem"},"title":"Elder audio interview (lang)","url":"https://coscrad.org/wp-content/uploads/2023/05/metal-mondays-mock2_370934__karolist__guitar-solo.mp3","mimeType":"audio/wav"}}
Attempting to execute command FSA: {"type":"PUBLISH_RESOURCE","payload":{"aggregateCompositeIdentifier":{"id":"dda1df6d-9ee4-4bd9-9a82-3d4bd74bb831","type":"mediaItem"}}}
Success
```

Reply:
Great.